iphone/TitaniumKit/TitaniumKit/Sources/API/KrollBridge.m

```diff
@@ -1366,7 +1366,7 @@ - (id)require:(KrollContext *)kroll path:(NSString *)path
 {
   NSURL *oldURL = [self currentURL];
   NSString *workingPath = [oldURL relativePath];
-  ResolvedModule *resolved = [self resolveRequire:path withWorkingPath:workingPath];
+  ResolvedModule *resolved = [[self resolveRequire:path withWorkingPath:workingPath] autorelease];
```

Comment:
Isn't it good if we use autorelease, where we are allocating the object. So better we should go inside function`-(ResolvedModule *)resolveRequire:(NSString *)path withWorkingPath:(NSString *)workingPath ` and use autorelease while allocating.